examples/main/main.cpp

```diff
@@ -139,7 +139,7 @@ int main(int argc, char ** argv) {
     // Add a space in front of the first character to match OG llama tokenizer behavior
     params.prompt.insert(0, 1, ' ');
 
-    std::string path_session = params.path_session;
+    std::string path_session    = params.path_prompt_cache;
```

Comment:
```suggestion
    std::string path_session = params.path_prompt_cache;
```
Oops, we missed a spacing error.